grove/alpha/simon/tests/test_utils.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+from grove.alpha.simon import utils as u
```

Comment:
newline between imports of different kinds PEP8
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
looks weird, but ok ..